src/components/Layout/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import Link from "next/link";
+
+import styles from "./Navbar.module.scss";
+
+const PAGES = [
+  { path: "/", label: "Home" },
+  {
+    path: "/members",
+    label: "Our Members",
```

Comment:
I don't think "Our" is required here, and shorter strings look better on navbars.

```suggestion
    label: "Members",
```

Reply:
Sure, I was copying the original site haha